index.html

```diff
@@ -0,0 +1,363 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta
+      name="viewport"
+      content="width=device-width, initial-scale=1, user-scalable=no"
+    />
+    <link rel="stylesheet" href="css/style.css" />
+    <title>Scout</title>
+  </head>
+  <body>
+    <header class="header">
+      <div class="container max-width">
+        <div class="empty-element none max-width"></div>
+        <div class="header-inner">
+          <a href="#" class="logo"> Skout </a>
+          <nav class="container-header-menu none">
+            <ul class="header-menu">
+              <li href="#" class="header-menu-item active-link">Why Skout</li>
+              <li href="#" class="header-menu-item">Products</li>
+              <li href="#" class="header-menu-item">Partners</li>
+              <li href="#" class="header-menu-item">Company</li>
+              <li href="#" class="header-menu-item">Tips&Tools</li>
+              <li href="#" class="header-menu-item">Blog</li>
+            </ul>
+
+            <div class="enter-buttons">
+              <button class="header-button">Get Skout</button>
+              <button class="header-button">Log In</button>
+            </div>
+          </nav>
+
+          <div class="hamburger-container">
+            <button class="hamburger-icon" id="hamburger"></button>
+          </div>
+        </div>
+
+        <div class="header-text"><span class="header-text-value">Tips & Tools</span></div>
+      </div>
+    </header>
+
+    <nav class="tabs">
+      <a href="#head-tips" class="head-tips current-tabs-link anchor ">Tips</a>
+
+      <a href="#head-glossary" class=" head-glossary anchor">Glossary</a>
+
+      <a href="#head-FAQs" class=" head-FAQs anchor">Faqs</a>
+    </nav>
+
+    <div class="accordions-container max-width">
+      <div class="skout-tips-container">
```

Comment:
Here also need to be a heading tag instead of div and change img to text. Also class name .scout-tips-container is too general, What if this text will be changed?